.circleci/config.yml

```diff
@@ -53,13 +53,13 @@ commands:
     steps:
       - macos/preboot-simulator:
           platform: "iOS"
-          version: "18.0"
+          version: "18.5"
           device: "iPhone 16"
 
 jobs:
   validate-code:
     macos:
-      xcode: 16.0.0 # Specify the Xcode version to use
+      xcode: 16.4.0 # Specify the Xcode version to use
```

Comment:
Can we also specify the resource class to the new upcoming default:

`resource_class: m4pro.medium`

like:
```yaml
jobs:
  validate-code:
    macos:
      xcode: 16.4.0 # Specify the Xcode version to use
    resource_class: m4pro.medium
```

Same for all the other jobs in this config

Tested in:
* https://github.com/adobe/aepsdk-messaging-ios/pull/427